codespell_lib/data/dictionary.txt

```diff
@@ -20166,6 +20166,7 @@ dissimialr->dissimilar
 dissimialrity->dissimilarity
 dissimialrly->dissimilarly
 dissimiar->dissimilar
+dissimilare->dissimilar, dissimilarly,
```

Comment:
Not sure about the `dissimilarly` suggestion. Have you ever seen such a typo out there?

Reply:
I have seen the similare typo and usually I am asked to add all variants of typos, so I added dissimilare as well.

Reply:
Do you mean you have seen `similare` as a typo of `similarly`? If so, the suggestion would obviously be OK. If not, I am not certain about it - but let's keep it!